src/game.js

```diff
@@ -209,6 +218,14 @@
             // reset the frame counter
             frameCounter = 0;
             frameRate = ~~(0.5 + 60 / me.sys.fps);
+            
+            // set step size based on the updatesPerSecond
+            stepSize = (1000 / Math.min(me.sys.updatesPerSecond, me.sys.fps));
```

Comment:
Yeah, my point with the last comment was 
```
me.sys.updatesPerSecond = 200;
me.sys.fps = 60;
```
...is an invalid scenario. `Math.min` was put into place intentionally to limit updatesPerSecond to 60 in that scenario.

Do you really need to update faster than you can possibly draw? I figured not, that's why I put that in.